### PR TITLE
Clarify the CLI installation instructions

### PR DESCRIPTION
**Pull request by valyo**
> **Before submitting the PR, please go through the sections below and fill in what you can. If there are any items that are irrelevant for the current PR, remove the row. If a relevant option is missing, please add it as an item and add a PR comment informing that the new option should be included into this template.**

> **All _relevant_ items should be ticked before the PR is merged**

# Description
Improvement in the MacOS/Linux executable part. Added links to Apple documentation about allowing apps to run. I think an extensive how-to with screenshots is out of the scope of this documentation. 
Small changes in the Windows executable part too, but lack of Windows computer for testing makes it hard to achieve something better.

- [x] Add a summary of the changes and the related issue
- [x] Add motivation and context regarding why the change is needed
- [x] Fixes DDS-1311

## Type of change

- [x] Documentation

# Checklist:

## Checks

- [x] Formatting: Black & Prettier checks pass
- [x] CodeQL passes
- [x] Tests
  - [x] The tests pass

### CODE REVIEW

**Comment by valyo** [comment]
for Windows we could maybe add something like this https://www.mathworks.com/matlabcentral/answers/94933-how-do-i-edit-my-system-path-in-windows
since the explanation of how to use the history etc is not really good.

File: docs/installation.rst [hunk 1]
```diff
@@ -65,19 +65,50 @@ PyPi
...
+   .. code-block:: bash
+
+      $ chmod +x dds-cli_ubuntu_x86_64   
+
+   or
+
+   .. code-block:: bash
+
+      $ chmod +x dds-cli_macos_x86_64   
+
```
Comment (inaod568) [inline docs/installation.rst:84]: Looks good but I think we should probably clarify that _if_ the file is not executable and they don't have permission, they need to do this.

**Review by inaod568 — CHANGES_REQUESTED** [review]

**Comment by inaod568** [comment]
> for Windows we could maybe add something like this https://www.mathworks.com/matlabcentral/answers/94933-how-do-i-edit-my-system-path-in-windows since the explanation of how to use the history etc is not really good.

@valyo Sorry, not following. Why do they need to change the system path? Or was this an example of a step by step thing regarding how to get the path to the executable? Like you've added now? 
I think what you've added now is definitely good enough and the rest will be up to them / the units to help them with. If there are complaints / suggestions saying that we should add more info then we can discuss it again 👍🏻

File: docs/installation.rst (hunk 1 above)
Comment (valyo) [reply docs/installation.rst:84]: My impression is that the file is never executable, and also, if we write smth like _if the file is not executable_ then we need to add instructions about how to check that. Isn't it much simpler to just give them the _chmod_ command.

Comment (inaod568) [reply docs/installation.rst:84]: Yeah true, just not sure we should encourage people to change permissions etc, but maybe I'm overly paranoid.

**Review by inaod568 — APPROVED** [review]
Looks good!

**Pull request merged**